smoke-tests/src/test/groovy/io/opentelemetry/smoketest/LibertySmokeTest.groovy

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.smoketest
+
+import io.opentelemetry.proto.trace.v1.Span
+import okhttp3.Request
+import spock.lang.Shared
+import spock.lang.Unroll
+
+import java.util.jar.Attributes
+import java.util.jar.JarFile
+
+class LibertySmokeTest extends AppServerTest {
+  //TODO run more configurations
+  @Shared
+  String jdk = 8
+  @Shared
+  String serverVersion = "20.0.0.12"
+
+  protected String getTargetImage(String jdk, String serverVersion) {
+    "ghcr.io/open-telemetry/java-test-containers:liberty-${serverVersion}-jdk$jdk-20201215.422527843"
+  }
+
+  boolean skipStartTarget() {
+    true
+  }
+
+  def setupSpec() {
+    startTarget(jdk, serverVersion)
+  }
+
+  def cleanupSpec() {
+    stopTarget()
+  }
+
+  @Unroll
+  def "#appServer test static file found on JDK #jdk"(String appServer, String jdk) {
+    String url = "http://localhost:${target.getMappedPort(8080)}/app/hello.txt"
+    def request = new Request.Builder().url(url).get().build()
+    def currentAgentVersion = new JarFile(agentPath).getManifest().getMainAttributes().get(Attributes.Name.IMPLEMENTATION_VERSION)
+
+    when:
+    def response = CLIENT.newCall(request).execute()
+    TraceInspector traces = new TraceInspector(waitForTraces())
+    Set<String> traceIds = traces.traceIds
+    String responseBody = response.body().string()
+
+    then: "There is one trace"
+    traceIds.size() == 1
+
+    and: "Response contains Hello"
+    responseBody.contains("Hello")
+
+    and: "There is one server span"
+    traces.countSpansByKind(Span.SpanKind.SPAN_KIND_SERVER) == 1
+
+    and: "Expected span names"
+    traces.countSpansByName('/app/hello.txt') == 1
```

Comment:
This span name indeed violates the spec's requirement for low cardinality. We have to review other app server instrumentations as well for this problem. /cc @vovencij

Reply:
As far as I know currently all servers behave that way. This is not the only scenario where span name has high cardianality /this-is-definitely-not-there-but-there-should-be-a-trace-nevertheless also is pretty much the same.

Reply:
Yep, and that also needs fixing...

Reply:
Changed span name to `HTTP {METHOD_NAME}`